packages/FieldTypes/src/components/Container/Container.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import PropTypes from "prop-types";
+import * as sc from "./Container.styles";
+import types from "../../types";
+
+export default function Container(props) {
+  const { children, align, color, ...moreProps } = props;
+
+  return (
+    <sc.Container $align={align} $color={color} {...moreProps}>
```

Comment:
Question: Are the `$` intentional?

Reply:
I'm also unfamiliar with this convention, I though I did notice a prop with the `$` in the `DataGrid`.

Reply:
if you pass color and align without the `$` will be spread on the final HTML dom, adding the `$` force the styledComponent component to drop it since is not a valid HTML attribute.

Reply:
Ahh.  Is that a common convention, or a Nahum convention?

Reply:
🙈 nahum's